### ISSUE TITLE
[Question] How to customize the loss calculation for PPO

### ISSUE THREAD

**Issue by olmoulin** [DLR-RM/stable-baselines3#2028]
### ❓ Question

Hi,
For a research purpose, I need to modify the loss calculation for PPO in order to take into account another parameter ... I have modified the PPO.py file as follow : loss = (policy_loss + self.ent_coef * entropy_loss + self.vf_coef * value_loss + gen_loss), gen_loss being my new parameter ... but this has absolutely no impact on the training  ... not sure, but am I making something wrong there ? thanks a lot in adance.

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] If code there is, it is [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014)
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by araffin** [comment]
Probably a duplicate of https://github.com/DLR-RM/stable-baselines3/issues/1881

**Comment by olmoulin** [comment]
This is what I have applied, and it does run properly my code, but it seems that no matter how much you modify the loss calculation it has no impact ... am I missing something ?

**Comment by araffin** [comment]
> This is what I have applied,

then this should be fine.

>  how much you modify the loss calculation it has no impact 

No impact on the behavior?
You should be able to log the total loss and see some difference (while fixing the seed).
